### PR TITLE
Hackmakers

### PR DESCRIPTION
**Pull request by kevinprince**

### CODE REVIEW

**Comment by kevinprince** [comment]
@cbetta @mseckington @leeky @nicwhatmore 

Can I get an OK this so its not late another week?

File: source/volunteer.html.erb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<section class="hackmakers">
+  <h1>Volunteers</h1>
+  <p>HackDays on the scale of HACKED are not be possible without an amazing team. Over next few weeks we will be posting here roles which we need filling for HACKED.</p>
```
Comment (mseckington) [inline source/volunteer.html.erb]: "are not be possible" -> drop the "be"

Comment (mseckington) [inline source/volunteer.html.erb]: "Over next few weeks" -> needs to be "Over the next few weeks"

File: source/volunteer.html.erb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  <p>All the roles are completely voluntary and un-paid however we will look after you and make sure you get regular breaks, meals and have a lot of fun! All volunteers will need to attend a HACKED staff briefing in Central London during the week of July 8th.</p>
+
+  <p>To volunteer for any role email <a href="mailto:help@hacked.io">help@hacked.io</a> and for technical roles include details of experience.</p>
+
+  <h4>Strategic Hacked Invitation Enrolment and Label Division (S.H.I.E.L.D)</h4>
+  <p>Ready to be the face of HACKED? You will be the first member of the team to greet attendees and be their first port of call during the weekend for any problems. Key responsibilities include:</p>
+  <ul>
+    <li>Registering attendees (issuing badges, fixing names etc)</li>
+    <li>Answering attendee questions (issues with internet, how do i get on the roof?)</li>
```
Comment (mseckington) [inline source/volunteer.html.erb]: "i" needs to be "I"

File: source/volunteer.html.erb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    <li>Keeping our front of house experience second to none.</li>
+  </ul>
+  <p><strong>Commitment:</strong> Flexible: Friday 10am-9pm, Saturday 8am to Sunday 8pm</p>
+
+  <h4>The Hacked International Response Supply Team (T.H.I.R.S.T)</h4>
+  <p>Your job is deliver refreshment at the speed of light! Know your coke from your diet pepsi? or which chocolate bar perfectly compliments bangers and mash you should signup for THIRST. Responsibilities include:</p>
```
Comment (mseckington) [inline source/volunteer.html.erb]: "Your job is deliver" -> needs to be "Your job is to deliver"

Comment (mseckington) [inline source/volunteer.html.erb]: "or which chocolate bar" needs to be "Or which chocolate bar"

File: source/volunteer.html.erb [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  <ul>
+    <li>Handing out ice-cold sodas</li>
+    <li>Keeping the sugary supply chain moving</li>
+    <li>Watching out for caffeine overload</li>
+  </ul>
+  <p><strong>Commitment:</strong> Flexible during following times: Friday 10am-9pm, Saturday 8am to Sunday 8pm</p>
+
+  <h4>Hacked Evolved Running and Operations Enlisted Section (H.E.R.O.E.S)</h4>
+  <p>Our support operations team will be the glue that holds together the entire event. Weather its sneaking beer into the venue, helping to move the giant H cube on to the stage or simply making sure everyone is fed and watered. Your skills will be put to the test in helping make the event run smoothly.</p>
```
Comment (mseckington) [inline source/volunteer.html.erb]: "Weather its sneaking beer" -> needs to be "Whether it's sneaking beer" 

(so fix "whether" and "it's")

File: source/volunteer.html.erb [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  
+  <h4>Register to Volunteer!</h4>
+    <iframe src="https://docs.google.com/forms/d/1JO3bcKuFMBjXbACDjFnhZBv65jboYWmNxF1ogs-cptM/viewform" frameborder="0" height="1000" width="100%" vspace="0" hspace="0" marginheight="5" marginwidth="5" scrolling="auto" allowtransparency="true"></iframe>
+</section>
+
+
+<section class="highlight">
+  <h1>FAQs?</h1>
+  <h4>Do volunteers get tickets to HACKED?</h4>
+  <p>Yes, all volunteers will get tickets to HACKED. However most roles have a minimum of approx 6 hour commitment during the weekend so your ability to hack something maybe slightly curtailed.</p>
```
Comment (mseckington) [inline source/volunteer.html.erb]: "maybe slightly curtailed" -> "may be slightly curtailed"

**Comment by kevinprince** [comment]
@mseckington changes done.

**Comment by kevinprince** [comment]
Going to put live today unless hear otherwise.

**Pull request merged**